mlrun/datastore/targets.py

```diff
@@ -1539,8 +1539,9 @@ def __init__(
                 # TODO: Remove this in 1.9.0
                 FutureWarning,
             )
-        if bootstrap_servers is not None:
-            attrs["brokers"] = brokers or bootstrap_servers
+            brokers = brokers or bootstrap_servers
```

Comment:
I would actually assert early on in the function that brokers and bootstrap_servers are not provided in the same constructor call - and raise an exception if not.
After that, we can simply do `brokers = bootstrap_servers`

Reply:
Done.